azure-devops/azext_devops/dev/common/services.py

```diff
@@ -136,93 +136,93 @@ def get_token_from_az_login(profile, user, tenant):
         return ""
 
 
-def _get_vss_connection(organization, credentials):
-    return VssConnection(get_base_url(organization), creds=credentials,
-                         user_agent='devOpsCli/{}'.format(VERSION))
+def _get_connection(organization, credentials):
+    return Connection(get_base_url(organization), creds=credentials,
+                      user_agent='devOpsCli/{}'.format(VERSION))
 
 
 def get_first_vss_instance_uri():
-    for key in _vss_connection:
+    for key in _connection:
         return key
 
 
 def get_release_client(team_instance=None):
-    connection = get_vss_connection(team_instance)
-    return connection.get_client(VSTS_MODULE + 'release.v4_0.release_client.ReleaseClient')
+    connection = get_connection(team_instance)
+    return connection.get_client(VSTS_MODULE + 'v5_0.release.release_client.ReleaseClient')
```

Comment:
In latest python client there is a concept of [released client factory](https://github.com/Microsoft/azure-devops-python-api/blob/dev/azure-devops/azure/devops/released/client_factory.py), can we use that instead of hard coding it here now?

Reply:
Released folder does not have all the clients. 

On using ClientFactory_v50 the issue is it tries to get the clients via 'azure.devops.v5_0' but since we are not using the pip package this again falls into the namespace clash where we will not get devops in azure.

Reply:
we should fix the function in factory in that case.
If we can't take it right now then also please open a task because sooner or later we should move to use this

Reply:
We can but more than the upside of removing hardcoding from our code, It has a downside of not having the freedom to replace the package content with the latest. I will open a task anyways for using the client factory.

Reply:
#509 Filed a task marking this as resolved.